am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/importer/DefaultPermissionIntegrationTest.java

```diff
@@ -40,7 +39,7 @@ void setUp() {
         resourceType = UUID.randomUUID().toString();
         service.addService(serviceName);
         service.addResourceDefinition(
-            resourceDefinition = createResourceDefinition(serviceName, resourceType, RESOURCE_NAME));
+            resourceDefinition = createResourceDefinition(serviceName, resourceType, UUID.randomUUID().toString()));
```

Comment:
with this addition I think we can remove the class level linelength supresssion :)

Reply:
👍